stable/jenkins/CHANGELOG.md

```diff
@@ -5,6 +5,17 @@ numbering uses [semantic versioning](http://semver.org).
 
 NOTE: The change log until version 1.5.7 is auto generated based on git commits. Those include a reference to the git commit to be able to get more details.
 
+## 1.11.1 Support multiple agents
+
+The README contains an example of using additional `agent` values to configure kubernetes pod templates in the default configuration as code.
+
+The pod template configuration based on `agent` values moved to a new template `jenkins.casc.podTemplate` with the following enhancements. 
+- `agent.command` quoted when not nil to support empty string (empty string ensures Jenkins doesn't apply a default value such as `/bin/sh -c`)
+- `agent.resources.limits`, `agent.resources.requests`, and `agent.yamlTemplate` have empty checks
+- `agent.volumes` added
+- `agent.workingDir` configurable
+- use of envVar as containerEnvVar is deprecated
+
```

Comment:
There is quire a lot going on in this PR. To make it easier for review I suggest to split it up into smaller ones. e.g. having one PR which just does:

> pod template configuration based on `agent` values moved to a new template `jenkins.casc.podTemplate`

And having the enhancements then in a separate?

Reply:
@torstenwalter I've simplified this PR as much as possible. I'll rebase once https://github.com/helm/charts/pull/21773 is merged.

Reply:
It's merged now. Thanks.

Reply:
@torstenwalter I've rebased.